draft-irtf-cfrg-hpke.md

```diff
@@ -316,7 +316,7 @@ operations, roles, and behaviors of HPKE:
 
 HPKE variants rely on the following primitives:
 
-* A Key Encapsulation Mechanism (KEM):
+* A key encapsulation mechanism (KEM):
```

Comment:
Looking at the [HTML](https://www.rfc-editor.org/authors/rfc9180.html) and the [PDF](https://www.rfc-editor.org/authors/rfc9180.pdf), the lists here got changed into definition lists. Semantically, that's probably what they should be. However, the rendering has no bullet points, which makes it kind of hard to read to me. Or maybe it is not the bullet points but the fact that `<code>` elements do _not_ have a slightly different color like in the rest of the document.

Reply:
I think the indentation is just due to things being at different levels.  In the main list, the first level is KEM/KDF/AEAD, and the functions are at the second level; the indentation is only at the second level.  The private key functions are at the first level, so they are not indented.

Nonetheless, I agree that this is more readable as a bulleted list.  Which is nice because I didn't reflect the definition list in this set of edits.  So I think we can leave this as it is, and negotiate with the RFC editor about reverting it to bullets.